sdk/monitor/azure-monitor-query/azure/monitor/query/aio/_metrics_batch_query_client_async.py

```diff
@@ -32,7 +32,7 @@ class MetricsBatchQueryClient:  # pylint: disable=client-accepts-api-version-key
         will be assumed. Defaults to 'https://metrics.monitor.azure.com'.
     """
 
-    def __init__(self, credential: AsyncTokenCredential, endpoint: str, **kwargs: Any) -> None:
+    def __init__(self, endpoint: str, credential: AsyncTokenCredential, **kwargs: Any) -> None:
```

Comment:
Is this only preview breaking? Worth calling out on the changelog, too

Reply:
Good catch! Missed this in the changelog. Added!